turbinia/workers/photorec.py

```diff
@@ -12,26 +12,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Task to extract binary files from an evidence object provided."""
+"""Task to run Photorec on disk images and retrieve deleted files ."""
 from __future__ import unicode_literals
 
 import logging
 import json
 import os
 
-from turbinia import config
+from turbinia import TurbiniaException
 from turbinia.workers import TurbiniaTask
-from turbinia.evidence import BinaryExtraction
 from turbinia.evidence import PhotorecOutput
 
 
 class PhotorecTask(TurbiniaTask):
 
-  def __init__(self):
-    super(PhotorecTask, self).__init__()
+  # def __init__(self):
```

Comment:
Can we just remove these lines instead of commenting them out?

Reply:
Done